_includes/sqlstyle.guide.md

```diff
@@ -45,41 +40,26 @@ Based on a work at [http://www.sqlstyle.guide][self].
 * Quoted identifiers—if you must use them then stick to SQL92 double quotes for
   portability (you may need to configure your SQL server to support this depending
   on vendor).
-* Object oriented design principles should not be applied to SQL or database
-  structures.
-
```

Comment:
it's not clear to me wtf they're talking about here, or how those examples tie in, without counterexamples. are they contrasting this with:

```
select hash from file_system where name = '.vimrc';
```
?